webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -45,32 +45,41 @@ class SakaiRubricsManager extends RubricsElement {
+      <div role="tabpanel" aria-labelledby="site-rubrics-title" id="site_rubrics">
+        <div class="rubric-title-sorting">
+          <div @click="${this.sortSiteRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site_name" key="site_name">site_name</sr-lang><span class="collpase-icon fa fa-chevron-up site_name sort_element_site"></span></a></div>
+          <div @click="${this.sortSiteRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site_title" key="site_title">site_title</sr-lang><span class="site_title sort_element_site"></span></a></div>
+          <div @click="${this.sortSiteRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="creator_name" key="creator_name">creator_name</sr-lang><span class="creator_name sort_element_site"></span></a></div>
+          <div @click="${this.sortSiteRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="modified" key="modified">modified</sr-lang><span class="modified sort_element_site"></span></a></div>
+          <div class="actions"><sr-lang key="actions">actions</sr-lang></div>
         </div>
+        <br>
+        <sakai-rubrics-list id="sakai-rubrics" @sharing-change="${this.handleSharingChange}" @copy-share-site="${this.copyShareSite}" token="Bearer ${this.token}"></sakai-rubrics-list>
+      </div>
 
         <div id="shared-rubrics-title" aria-expanded="${this.sharedRubricsExpanded}" role="tab" aria-multiselectable="true" class="manager-collapse-title" title="${tr("toggle_shared_rubrics")}" tabindex="0" @click="${this.toggleSharedRubrics}">
           <div>
             <span class="collpase-icon fa fa-chevron-down"></span>
             <sr-lang key="shared_rubrics">shared_rubrics</sr-lang>
           </div>
-          <div class="hidden-xs"><sr-lang key="site_title">site_title</sr-lang></div>
-          <div class="hidden-xs"><sr-lang key="creator_name">creator_name</sr-lang></div>
-          <div class="hidden-xs"><sr-lang key="modified">modified</sr-lang></div>
-          <div class="actions"><sr-lang key="actions">actions</sr-lang></div>
         </div>
 
-        <div role="tabpanel" aria-labelledby="shared-rubrics-title" id="shared_rubrics">
-          <div id="sharedlist">
-            <sakai-rubrics-shared-list token="Bearer ${this.token}" id="sakai-rubrics-shared-list" @copy-share-site="${this.copyShareSite}" ></sakai-rubrics-shared-list>
-          </div>
+      <div role="tabpanel" aria-labelledby="shared-rubrics-title" id="shared_rubrics">
+        <div id="sharedlist">
+          <div class="rubric-title-sorting">
+          <div @click="${this.sortSharedRubrics}"><a href="javascript:void(0)" style="text-decoration: none;"><sr-lang class="site_name" key="site_name">site_name</sr-lang><span class="collpase-icon fa fa-chevron-up site_name sort_element_shared"></span></a></div>
```

Comment:
Everywhere else in the rubrics code, I think, we use hyphens for the class name separators. Could you stick to the same convention?